### PR TITLE
Call Python from R through Rcpp

### PR DESCRIPTION
**Pull request by wush978**
Dear @jjallaire, @eddelbuettel,

Here is a post demonstrating how to call python from R through Rcpp.

### CODE REVIEW

File: _posts/2014-04-06-rcpp-python.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
+---
+title: Call Python from R through Rcpp
+author: Wush Wu
+license: GPL (>= 2)
+tags: python integration
```
Comment (eddelbuettel) [inline _posts/2014-04-06-rcpp-python.md:5]: We try not add new tags with new posts.  Let's try reusing existing tags first.  Can you change this please?

File: _posts/2014-04-06-rcpp-python.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+
+## Introduction
+
+This is a brief introduction of calling python from R through Rcpp. [The official documentation of python](https://docs.python.org/2/extending/embedding.html) explains how to embed python into C/C++ application. Moreover, the [Boost.Python](http://www.boost.org/doc/libs/1_55_0/libs/python/doc/) library provides seamless interoperability between C++ and the Python programming language. Similarlly, Rcpp provides interoperability between C++ and R. Therefore, it is not hard to call python from R through Rcpp and Boost.Python.
+
+Although there is a package [*rPython*](http://cran.r-project.org/web/packages/rPython/index.html) which provides an interface of python in R through Java, it is interesting to try to connect them through C++.
+
+In this article, I'll show you how to call python 2.7 from R on Ubuntu. Please install the following packages:
+
```
Comment (eddelbuettel) [inline _posts/2014-04-06-rcpp-python.md:21]: I guess you need to fill in the list of packages.  Or maybe remove the half-sentence 'Please install ...' here.

(Oh, and sorry -- edit the .Rmd below, not this one.)

**Comment by eddelbuettel** [comment]
Very good. I once worked on that too and had a working minimal prototype but then somehow lost interest / didn't continue this.   I'll review more later, but added two simple comments.

**Comment by eddelbuettel** [comment]
I am doing the edits by hand now.  A lot of s/python/Python/ and small stuff.

Great post though -- congrats.  Gave it 'featured' tag.

File: _posts/2014-04-06-rcpp-python.md [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+The most difficult thing is to establish a development environment. To build an embeded python, we need to install the following package:
+
+```
+sudo apt-get install python2.7 python2.7-dev libboost-python-dev
+```
+
+Then, we should pass the following flags to the compiler:
+
+
+{% highlight r %}
```
Comment (jjallaire) [inline _posts/2014-04-06-rcpp-python.md:34]: A good candidate for Rcpp::plugin!

**Comment by eddelbuettel** [comment]
Yes indeed!

Can you test this on OS X ?  Or do you fear a rat's nest of dependencies?

(Final version with a bunch of stylistic edits now posted on-site.)

**Pull request closed without merging**